src/main/java/mate/academy/dao/BookDaoImpl.java

```diff
@@ -0,0 +1,109 @@
+package mate.academy.dao;
+
+import java.math.BigDecimal;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Dao;
+import mate.academy.model.Book;
+
+@Dao
+public class BookDaoImpl implements BookDao {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/my_database";
+    private static final String DB_USER = "admin";
+    private static final String DB_PASSWORD = "admin";
+
+    @Override
+    public Book create(Book book) {
+        String query = "INSERT INTO books (title, price) VALUES (?, ?)";
+        try (Connection connection = DriverManager.getConnection(DB_URL, DB_USER, DB_PASSWORD);
```

Comment:
Consider using a connection pool instead of creating a new connection for each operation. This can improve performance and resource management.